app/src/main/java/com/nextcloud/talk/adapters/messages/PreviewMessageViewHolder.kt

```diff
@@ -128,7 +128,7 @@ abstract class PreviewMessageViewHolder(itemView: View?, payload: Any?) :
                     )
                 }
                 clickView!!.setOnLongClickListener { l: View? ->
-                    onMessageViewLongClick(message)
+                    previewMessageInterface!!.onPreviewMessageLongClick(message)
```

Comment:
as the method 
`private fun onMessageViewLongClick(message: ChatMessage)..`
is no longer used it can be deleted.
https://github.com/nextcloud/talk-android/pull/3319/files#diff-d097b26ec3d15c322bf4cf32bdb89ba37a00fd8c41bde2109848e68b0e559729R283

Reply:
Done